### PR TITLE
Vadym Horval HW5

### PR DESCRIPTION
**Pull request by VadymHorval**

### CODE REVIEW

File: horval_hw/vadym_horval_hw5/horval_hw5_task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Homework 5 task 1 Vadym Horval
+import random
+
+attempts = 0
+entered_num = 101
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task1.py:5]: Nice Point

File: horval_hw/vadym_horval_hw5/horval_hw5_task1.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    if entered_num < generated_num:
+        print("Your number is less")
+    elif entered_num > generated_num:
+        print("Your number is higher")
+    else:
+        print("You WON!")
+    attempts +=  1
+
+if entered_num != generated_num:
+     print("You didn't guess the number:", generated_num)
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task1.py:19]: Good Decision

File: horval_hw/vadym_horval_hw5/horval_hw5_task2.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+sum = 0
+
+print("Please, enter ten natural numbers greater than 2")
+
+for i in range(10):
+    entered_num = int(input("Please, enter number:"))
+    if not entered_num % 5:
+        sum += 1
+
+print("Numbers that are multiples of 5:",sum)
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task2.py:11]: Good Decision.

File: horval_hw/vadym_horval_hw5/horval_hw5_task3.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+for i in range(10):
+    line = ""
+    for a in range(10):
+        if a != 0 and i != 0:
+            line += str((i)*(a)) + " "
+        if len(str((i)*(a))) == 1:
+            line += " "
+    print(line)
+
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task3.py:12]: Please delete this empty line

File: horval_hw/vadym_horval_hw5/horval_hw5_task3.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Homework 5 task 3 Vadym Horval
+print("Multiplication table")
+
+for i in range(10):
+    line = ""
+    for a in range(10):
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task3.py:6]: Just advise: it's good practice to use i, j, k as iterable variables, not a

File: horval_hw/vadym_horval_hw5/horval_hw5_task4.py [hunk 6]
```diff
@@ -0,0 +1,11 @@
+ #Homework 5 task 4 Vadym Horval
+
+for i in range(10):
+    line = ""
+    for a in range(20):
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task4.py:5]: The same as previous

File: horval_hw/vadym_horval_hw5/horval_hw5_task4.py [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+        if a != 0 and i != 0 and i != 9:
+            line += "+"
+        else:
+            line += "*"  
+    line += "*"
+    print(line)
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task4.py:11]: Good Decision.

File: horval_hw/vadym_horval_hw5/horval_hw5_task5.py [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+        suma += entered_num
+    elif entered_num > number_H:
+        product *= entered_num
+    elif entered_num != number_P and entered_num != number_H:
+        count += 1
+
+print("The sum of numbers less than", number_P,":", suma)
+print("The product of numbers greater than", number_H,":", product)
+print("Number of numbers in range from", number_P,"to", number_H,"is", count)
+
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task5.py:26]: Good Decision, but delete these empty lines, please.

File: horval_hw/vadym_horval_hw5/horval_hw5_task6.py [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+    
+    entered_num = int(input("Please, enter number:"))
+    
+    if entered_num < 0:
+        negative_counter += 1
+    elif entered_num > 0:
+        positive_counter += 1
+    
+
+print("Percent of negative numbers", negative_counter*100/(positive_counter+negative_counter),"Percent of posiyive numbers", positive_counter*100/(negative_counter+positive_counter))
```
Comment (katemalash) [inline horval_hw/vadym_horval_hw5/horval_hw5_task6.py:19]: Good Decision.

**Review by katemalash — APPROVED** [review]
Home work is done good. You can merge this pull-request.

**Review by VadymHorval — COMMENTED** [review]
followed all the recommendations

**Pull request merged**